plugins/destination/snowflake/client/client_test.go

```diff
@@ -1,19 +1,24 @@
 package client
 
 import (
-	"os"
 	"testing"
 
 	"github.com/cloudquery/plugin-sdk/plugins/destination"
 )
 
 func TestPlugin(t *testing.T) {
+<<<<<<< HEAD
 	p := destination.NewPlugin("snowflake", "development", New)
+=======
+	p := destination.NewPlugin("snowflake", "development", New, destination.WithManagerWriter())
+>>>>>>> da4dc1c6c (feat(destinations): Migrate to managed batching SDK)
 	destination.PluginTestSuiteRunner(t, p,
 		Spec{
-			ConnectionString: os.Getenv("SNOW_TEST_DSN"),
+			// ConnectionString: os.Getenv("SNOW_TEST_DSN"),
+			// REMBAZX.ZO72963
+			ConnectionString: "yevgenyp:tur*jup@prm8ETC!uwk@yk49522.europe-west4.gcp/testdb/public?warehouse=test",
```

Comment:
Probably don't want this here (and this should be rotated now ;) )

Reply:
oouch. rotated :)